TSC-Charter.md

```diff
@@ -176,10 +187,71 @@ discussion will continue.
 For all votes, a simple majority of all TSC members for, or against, the issue
 wins. A TSC member may choose to participate in any vote through abstention.
 
+While the results of all votes must be made public, the actual individual
+ballots cast for most votes may be made public or confidential at the discretion
+of the TSC Chair. However, individual ballots for recertification or removal of
+TSC members must remain confidential. The names of members voting or abstaining
+in all votes must be made public.
+
+Following the completion of all votes, a public statement must be made via
+GitHub specifying the results of the vote. Contextual detail about why the
+vote was held, including a listing of the specific questions voted on must be
+included in the statement.
+
 Note that, in addition to requiring a simple majority vote of the TSC, all
 changes to this charter are also subject to approval from the Node.js
 Foundation board.
 
+### Votes to Remove Members
+
+Requests to remove a member from the TSC may be raised only by other members of
+the TSC or by members of the Node.js Foundation Board. When an issue to remove
+someone from the TSC is raised, the issue must include a list of the specific
+issues the TSC is being asked to consider as grounds for removal. The discussion
+and vote will be limited strictly to considering of the specific issues listed
+in the issue. The issue must also include a description of what outcome those
+reporting the issue expect.
+
+The vote must be structured in terms of whether or not the specific issues
+listed constitute grounds for removal.
+
+When votes to remove individuals from the TSC are scheduled, the member in
+question is to be notified about the vote in advance along with adequate
+contextual details of why the vote is taking place, what, if any, specific
+complaints are being leveled, and the specific questions that the members are
+being asked to consider. The member in question shall be given no less than one
+week before the vote to provide an answer or rebuttal to those issues.
+
+Neither the member in question, nor the member or members raising the issue
```

Comment:
Would an option be to allow both the raisers and the member in question.  If its one raiser then they will balance out, otherwise more raisers end up with more weight ?